os/include/sys/boardctl.h

```diff
@@ -58,6 +58,7 @@
  ****************************************************************************/
 #include <tinyara/config.h>
 #include <stdint.h>
+#include <stdlib.h>
```

Comment:
Why do you need to add this for this commit?

Reply:
`stdlib.h` is needed for `EXIT_SUCCESS`.